src/main/kotlin/nl/avisi/structurizr/site/generatr/site/RelativeUrl.kt

```diff
@@ -3,7 +3,7 @@ package nl.avisi.structurizr.site.generatr.site
 import java.nio.file.Path
 import kotlin.io.path.relativeTo
 
-fun String.asUrlRelativeTo(otherUrl: String) =
+fun String.asUrlRelativeTo(otherUrl: String, appendSlash: Boolean = true) =
```

Comment:
Instead of the `Boolean` parameter, I prefer to have 2 functions with a name that indicates the reason for adding a backslash or not. That way, this abstraction doesn't "leak" implementation details about how links should be formatted. The `true` case could be something like `fun String.asUrlToDirectory(relativeTo: String)...` and the `false` case could be something like `fun String.asUrlToFile(relativeTo: String)...`.